frame/executive/src/lib.rs

```diff
@@ -615,7 +615,7 @@ mod tests {
 				header: Header {
 					parent_hash: [69u8; 32].into(),
 					number: 1,
-					state_root: hex!("489ae9b57a19bb4733a264dc64bbcae9b140a904657a681ed3bb5fbbe8cf412b").into(),
+					state_root: hex!("409fb5a14aeb8b8c59258b503396a56dee45a0ee28a78de3e622db957425e275").into(),
```

Comment:
Why have this changed?

Reply:
Previous implementation of in memory backend used `trie_root` function which allows empty values in the trie. `TrieDBMut` that's used by the regular backend skips all empty values, hence storage root is different.